execution/executor/src/components/chunk_output.rs

```diff
@@ -317,6 +310,14 @@ pub fn update_counters_for_processed_chunk<T, O>(
                     } else if *discard_status_code == StatusCode::TRANSACTION_EXPIRED {
                         "discard_transaction_expired"
                     } else {
+                        // Only log if it is an interesting discard
+                        sample!(
```

Comment:
I think discard is already interesting if it's not because of sequence number?

Reply:
it was a sample above, which lead it to be only on sequence numbers. I kept it as sample (with same frequency), just moved only when it is something interesting. 

comment here is explanation on why is it logging only in else clause.

we cannot log all discards, in case people start submitting discards with high frequency